paparazzi/src/main/java/app/cash/paparazzi/internal/ImageUtils.kt

```diff
@@ -114,33 +114,45 @@ internal object ImageUtils {
     }
     assertEquals(TYPE_INT_ARGB.toLong(), goldenImage.type.toLong())
 
-    val imageWidth = Math.min(goldenImage.width, image.width)
-    val imageHeight = Math.min(goldenImage.height, image.height)
+    val goldenImageWidth = goldenImage.width
+    val goldenImageHeight = goldenImage.height
+
+    val imageWidth = Math.max(goldenImageWidth, image.width)
+    val imageHeight = Math.max(goldenImageHeight, image.height)
```

Comment:
Might it be clearer to name these differently? e.g.:
```suggestion
    val deltaWidth = Math.max(goldenImageWidth, image.width)
    val deltaHeight = Math.max(goldenImageHeight, image.height)
```